src/helpers/helpers.js

```diff
@@ -950,6 +950,27 @@ module.exports.register = function (Handlebars, options, params) {
         },
 
         /**
+        * Consistently generate the edit link for a file in VSTS
+        **/
+        editInVSTSLink: function (options) {
+            var src = options.hash.src || (typeof this.page !== 'undefined' ? this.page.src : '');
+            encodeURIComponent(src);
+            return [
+                stache.config.vsts_protocol,
+                stache.config.vsts_base,
+                '/',
+                stache.config.vsts_project,
+                '/_git/',
+                stache.config.vsts_repo,
+                '?path=',
+                src,
```

Comment:
`encodeURIComponent` returns a new string, so you'll need to move line 957 down to here.